docs/slurk_prerequisites.rst

```diff
@@ -135,7 +144,7 @@ Checkout and run slurk
 
     $> ssh-keygen
 
-2. Upload or copy the generated public key to your github SSH settings
+2. Upload or copy the generated public key to your `github SSH settings <https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account>`_
```

Comment:
Why is an ssh key required to run slurk? Why cloning is required at all? I'd just run the docker container with some environment variables and volumes passed. Maybe we want to split the section into "Run with docker (recommended)" and "Run without docker".

Reply:
You run the docker container with the provided script, so you would need that from the repository at least. I assume some would also want to add custom plugin scripts for which they may have to slightly alter slurk. I don't think having them clone slurk is a bad idea. I don't get the feeling that slurk is something you just use as it is.

Reply:
I assume anyone who has write access to this repository knows how to push to git. For any other, `https` could be used:
```bash
git clone https://github.com/clp-research/slurk
```